### PR TITLE
[API] Add ref to create/edit issue options & deprecated assignee

### PR DESCRIPTION
**Pull request by 6543**
as title + some docs updates

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: modules/structs/issue.go [hunk 1]
```diff
@@ -50,8 +50,9 @@ type Issue struct {
 	Ref              string     `json:"ref"`
 	Labels           []*Label   `json:"labels"`
 	Milestone        *Milestone `json:"milestone"`
-	Assignee         *User      `json:"assignee"`
-	Assignees        []*User    `json:"assignees"`
+	// deprecated
```
Comment (jolheiser) [inline modules/structs/issue.go:53]: Can you mention this in the description and/or title?

**Review by jolheiser — COMMENTED** [review]
Just a nit to mention unrelated changes

File: modules/structs/issue.go (hunk 1 above)
Comment (6543) [reply modules/structs/issue.go:53]: @jolheiser done

**Review by jolheiser — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**